roles/neutron-common/tasks/main.yml

```diff
@@ -108,3 +108,12 @@
   tags:
     - logrotate
     - logging
+
+- name: change neutron log file permissions
+  file: path={{ item }} mode=0644 owner=neutron group=neutron state=touch
+  with_items:
+    - /var/log/neutron/neutron-server.log
+    - /var/log/neutron/neutron-l3-agent.log
+    - /var/log/neutron/neutron-linuxbridge-agent.log
+    - /var/log/neutron/neutron-dhcp-agent.log
+    - /var/log/neutron/neutron-metadata-agent.log
```

Comment:
lbaas is added (optionally) with #1190  and specifically https://github.com/blueboxgroup/ursula/pull/1190/files#diff-dc497c30f745d0a73efc2f57e6e706a0R87